src/main/java/one/util/streamex/StreamEx.java

```diff
@@ -1764,11 +1764,6 @@ public void forPairs(BiConsumer<? super T, ? super T> action) {
      * <p>
      * This is a <a href="package-summary.html#StreamOps">quasi-intermediate
      * operation</a>.
-     * 
-     * <p>
-     * The size of the resulting stream is one element less than the input
```

Comment:
Thanks, indeed this should be changed. I forgot to update this part in version 0.6.4 when semantics of withFirst changed. However, it's better to say that the size remains the same than removing this completely. I will apply this change separately.